src/index.js

```diff
@@ -0,0 +1,47 @@
+const CryptoJS = require('ts.cryptojs256');
+
+/**
+ * Return a base64 URL
+ *
+ * @param {string} data - some data to be base64 encoded
+ * @return {string} A base64url encoded string
+ */
+function base64url (data) {
+  return CryptoJS.enc.Base64
+    .stringify(data)
+    .replace(/=+$/, '')
+    .replace(/\+/g, '-')
+    .replace(/\//g, '_');
+}
+
+/**
+ * Create a very basic JWT signature
+ *
+ * @param {Object} data - the data object you want to have signed
+ * @param {string} secret - secret to use to sign token with
+ * @param {Object} options - JWT header options
+ * @return {string} JSON Web Token that has been signed
+ */
+function sign (data, secret, options = {}) {
+  const defaultOptions = {
```

Comment:
Maybe define the defaults at the top of the file (rather than here in the function)? You also don't really need a default for `options` if you are only going to use it as a non-first argument for `Object.assign`.

```js
const defaultHeader = { alg: 'HS256', typ: 'JWT' };

function sign (data, secret, options) {
  const header = Object.assign({}, defaultHeader, options);
```